packages/evm/test/disputes/disputes-settle-round.js

```diff
@@ -294,8 +294,8 @@ contract('DisputeManager', ([_, drafter, appealMaker, appealTaker, guardian500,
                     assertBn(actualWeight, weight, 'guardian weight should not have changed')
 
                     const { available } = await protocolHelper.guardiansRegistry.balanceOf(address)
-                    const expectedANJReward = expectedCollectedTokens.mul(bn(weight)).div(bn(expectedCoherentGuardians))
-                    const expectedCurrentAvailableBalance = previousBalances[address].available.add(expectedANJReward)
+                    const expectedRedistributionReward = expectedCollectedTokens.mul(bn(weight)).div(bn(expectedCoherentGuardians))
```

Comment:
Thought `expectedRedistributionReward` would be better than `expectedANTReward`, but let me know if it doesn't express this concept well (I saw it as a redistribution of locked juror stakes, from the losing side to the winning side).